creation/web_base/setup_x509.sh

```diff
@@ -377,98 +386,118 @@ get_trust_domain() {
+                cred_updated+=idtoken,
+            else
+                warn "$out"
+            fi
+        fi
     else
-        warn "$out"
+        warn "$ERROR"
     fi
-else
-    warn "$ERROR"
-fi
 
-# x509 - skip if there is no X509_USER_PROXY
-if ! X509_USER_PROXY=$(get_x509_proxy); then
-    warn "Skipping x509, X509_USER_PROXY not available: $X509_USER_PROXY"
-    unset X509_USER_PROXY
-else
-    if out=$(get_x509_certs_dir); then
-        export X509_CERT_DIR="$out"
-        if out=$(check_x509_tools); then
-            [[ -n "$out" ]] && warn "$out"
-            if out=$(safe_copy_and_protect "$X509_USER_PROXY" "$gwms_credentials_dir/myproxy"); then
-                # Copy successful, set env variables
-                export X509_USER_PROXY_ORIG="$X509_USER_PROXY"
-                export X509_USER_PROXY="$out"
-                if out=$(get_x509_expiration "$X509_USER_PROXY"); then
-                    # Copy succesfull and proxy valid, set values in glidein_config
-                    gconfig_add X509_CERT_DIR   "$X509_CERT_DIR"
-                    gconfig_add X509_USER_PROXY "$X509_USER_PROXY"
-                    gconfig_add X509_USER_PROXY_ORIG "$X509_USER_PROXY_ORIG"
-                    gconfig_add X509_EXPIRE  "$out"
-                    cred_updated+="proxy,"
+    # x509 - skip if there is no X509_USER_PROXY
+    if ! X509_USER_PROXY=$(get_x509_proxy); then
+        warn "Skipping x509, X509_USER_PROXY not available: $X509_USER_PROXY"
+        unset X509_USER_PROXY
+    else
+        if out=$(get_x509_certs_dir); then
+            export X509_CERT_DIR="$out"
+            if out=$(check_x509_tools); then
+                [[ -n "$out" ]] && warn "$out"
+                if out=$(safe_copy_and_protect "$X509_USER_PROXY" "$gwms_credentials_dir/myproxy"); then
+                    # Copy successful, set env variables
+                    export X509_USER_PROXY_ORIG="$X509_USER_PROXY"
+                    export X509_USER_PROXY="$out"
+                    if out=$(get_x509_expiration "$X509_USER_PROXY"); then
+                        # Copy succesfull and proxy valid, set values in glidein_config
+                        gconfig_add X509_CERT_DIR   "$X509_CERT_DIR"
+                        gconfig_add X509_USER_PROXY "$X509_USER_PROXY"
+                        gconfig_add X509_USER_PROXY_ORIG "$X509_USER_PROXY_ORIG"
+                        gconfig_add X509_EXPIRE  "$out"
+                        cred_updated+="proxy,"
+                    else
+                        warn "$out"
+                    fi
                 else
                     warn "$out"
                 fi
             else
+                # "$error_gen" -error "setup_x509.sh" "WN_Resource" "$STR"
                 warn "$out"
             fi
         else
-            # "$error_gen" -error "setup_x509.sh" "WN_Resource" "$STR"
+            #"$error_gen" -error "setup_x509.sh" "WN_Resource" "$STR1" "directory" "$X509_CERT_DIR"
```

Comment:
same comment about replacing the script name with `$0`

Reply:
The `setup_x509.sh` in this line might have been forgotten to rename the script to SCRIPT_NAME